SPECS-EXTENDED/zopfli/zopfli.spec

```diff
@@ -0,0 +1,134 @@
+%global __cmake_in_source_build 1
+%global so_ver 1.0.3
```

Comment:
nit: we can get rid of so_ver and use %version instead

Reply:
Thank you for the review and approval. I have implemented your review comment in the recent commits.  Ran buddy builds after that. Here are the results. Both of the builds passed.
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=295357&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=295358&view=results)